test/run_tests.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
```

Comment:
How about using BATS here, like we do with checkpointctl?

Reply:
@snprajwal here each test function needs to be called in a particular order, and they depend on each other to pass all the test cases. I could write them to be independent, but I want to test how our operator behaves during policy updates and the creation of new checkpoints. So the question is, does BATS work with this properly? I think BATS runs test functions separately and starts in a clean state.

Reply:
should we write a single BATS test that includes multiple steps?

Reply:
It is not a good practice for tests to depend on each other because a failure in one test may cause other tests to fail unexpectedly. This makes it difficult to identify the root cause of the failure.

It is generally better to design the tests to be independent and isolated from each other. Each test should set up its own environment and clean up afterwards (see `setup()` and `teardown()`):

https://bats-core.readthedocs.io/en/stable/writing-tests.html